packages/swingset-liveslots/src/virtualObjectManager.js

```diff
@@ -260,15 +267,15 @@ const makeFacets = (
 };
 
 const insistDurableCapdata = (vrm, what, capdata, valueFor) => {
-  capdata.slots.forEach((vref, idx) => {
+  for (const [idx, vref] of entries(capdata.slots)) {
     if (!vrm.isDurable(vref)) {
       if (valueFor) {
-        Fail`value for ${what} is not durable: slot ${q(idx)} of ${capdata}`;
+        Fail`value for ${what} is not durable: slot ${b(idx)} of ${capdata}`;
       } else {
-        Fail`${what} is not durable: slot ${q(idx)} of ${capdata}`;
+        Fail`${what} is not durable: slot ${b(idx)} of ${capdata}`;
       }
     }
-  });
+  }
```

Comment:
I think the PR would be clearer if the hygiene/cleanup changes and the substantive changes were in separate commits.

(That said, I very much like the legibility improvement from the deprecation of `forEach` in favor of `for...of` loops. It's a welcome respite from what seems like our general trend towards favoring more FP-like coding idioms, which I find much, much harder to read.)

Reply:
Done at https://github.com/Agoric/agoric-sdk/pull/8013